olive/systems/azureml/aml_system.py

```diff
@@ -92,6 +97,27 @@ def _create_environment(self, docker_config: AzureMLDockerConfig):
             return Environment(image=docker_config.base_image, conda_file=docker_config.conda_file_path)
         raise ValueError("Please specify DockerConfig.")
 
+    def _create_compute(self, aml_compute_config: AmlComputeConfig):
+        logger.info(f"Creating compute {aml_compute_config.name}...")
+        ml_client = self.azureml_client_config.create_client()
+        try:
+            from azure.ai.ml.entities import AmlCompute
+
+            cluster_basic = AmlCompute(
+                name=aml_compute_config.name,
+                type="amlcompute",
+                size=aml_compute_config.size,
+                location=aml_compute_config.location,
+                min_instances=aml_compute_config.min_instances,
+                max_instances=aml_compute_config.max_instances,
+                idle_time_before_scale_down=aml_compute_config.idle_time_before_scale_down,
+            )
+            ml_client.begin_create_or_update(cluster_basic).result()
```

Comment:
If the compute with the same name exists, we will get an error here and throw OliveSystemException.

Reply:
I may think that: 
1. every computer created by Olive should be shutdown properly after the optimization done
2. If there is the same name, it means the aml compute instance is exactly same with the existed one. Why we throw the OliveSystemException?
3. How to reuse the exited aml compute:
     - if the existed compute is running with other optimization job (should wait or throw the error?
     - if the existed compute is not activated, we can directly leverage that?

Reply:
The original thought is, the functionalities of `aml_compute` and `aml_compute_config` are different. The `aml_compute` config means the user wants to use the existing one, and `aml_compute_config` means the user wants to create a new one with user-provided name, SKU, etc. If the compute name exists in `aml_compute_config` and reusing that compute with `aml_compute_config`, this will confuse the boundaries between these 2 configurations. (`aml_compute_config` this term may confuse the user unless we have a clear documentation about it).

But I agree we should consider shutting it down or deleting it after the workflow run.